### DISCUSSION TITLE
Is there tracking-server vs python-client version compatibility table?

### DISCUSSION THREAD

**Discussion by anencore94** [mlflow/mlflow#7742] [Q&A]
Hi forks, I've been using mlflow tracking server for a long time, so I'm using v1.23.0 for my tracking server now.
I also used my mlflow python client version to v1.23.0.

Recently I've tried to use mlflow tracking server v2.0 with the same client (v1.23.0), but some method doesn't work. 
So I have to downgrade mlflow tracking server to v1.23.0 again.

I try to find the version compatiblility of server-client compatibility. However it was hard to find any.
Is there any official compatibility table ? Please let me know.
Thanks in advance :)

---
### Similar Issues
- #5171

**Comment by JHibbard** [comment]
Hi anencore94,

Thanks for bringing up this issue. There's not a compatibility matrix that I know of, but this does look like a frustrating issue. I'll see if there's anything that can be done to generate a compatibility matrix as a documentation fix. I'm unsure how difficult that will be though.

Jim

**Reply by anencore94** [reply]
Thanks for checking this issue, let me know if there is any updates on this issue, or some clue to generate this matrix. @JHibbard
